app/controllers/curiosity_cards_controller.rb

```diff
@@ -1,7 +1,8 @@
 class CuriosityCardsController < ApplicationController
   before_action :authenticate_user!, except: [:index, :show]
-  before_action :find_curiosity, only: [:show, :edit]
-  before_action :find_curiosity_scope_by_current_user, only: [:update, :destroy]
+  before_action :find_curiosity, only: [:show, :edit, :update, :destroy]
+
+  after_action :verify_authorized, except: :index
```

Comment:
Is this actually working?

Reply:
yes and not xd, I need to refactor this function but yeah, it's still working with some methods hehe